test/scrabble_test.rb

```diff
@@ -5,9 +5,20 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+  skip
+  def test_it_exists
+    scrabble = Scrabble.new
+  end
+
+
   def test_it_can_score_a_single_letter
     skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
```

Comment:
This is our most simple case. I would work to make this pass before I moved on to trying to score a whole word.

Start with smaller bites and build on them:

1) Can I score a single letter?
    1a) Can I score "a"?
    1b) Can I score "f"?
2) Can I score an empty string?
3) Can I score `nil`?
4) Can I score "hello"?

I don't want you to really think about 2, 3, or 4 until you have 1 working.